pkg/rtc/types/interfaces.go

```diff
@@ -285,6 +285,7 @@ type LocalParticipant interface {
 	IsDisconnected() bool
 	IsIdle() bool
 	SubscriberAsPrimary() bool
+	GetClientInfo() *livekit.ClientInfo
 	GetClientConfiguration() *livekit.ClientConfiguration
```

Comment:
`ClientConfiguration.ResumeConnection` was intended to capture migrations. should we use ClientConfigurations instead?